sponsors.html

```diff
@@ -178,141 +178,263 @@ <h1 style="font-size:50px">CURRENT SPONSORS</h1>
-              <img class="img-rounded" src="img/sponsor/uw_cs.png"/>
-            </a>
-            <a class="tile" href="https://uwaterloo.ca/civil-environmental-engineering/" target="_blank">
-              <img class="img-rounded" src="img/sponsor/uw_civil_and_environmental.png"/>
-            </a>
-            <a class="tile" href="http://www.patternsthings.com/" target="_blank">
-              <img class="img-rounded" src="img/sponsor/patterns_n_things.png"/>
-            </a>
-            <a class="tile" href="http://www.weef.uwaterloo.ca/index.php" target="_blank">
-              <img class="img-rounded" src="img/sponsor/uw_weef.png"/>
-            </a>
-            <a class="tile" href="http://www.marsland.ca/" target="_blank">
-              <img class="img-rounded" src="img/sponsor/marsland.png"/>
-            </a>
-            <a class="tile" href="http://miltera.com/" target="_blank">
-              <img class="img-rounded" src="img/sponsor/miltera.png"/>
-            </a>
-            <a class="tile" href="http://www.samuel.com/en/Pages/default.aspx" target="_blank">
-              <img class="img-rounded" src="img/sponsor/samuel_son_and_co.png"/>
-            </a>
-            <a class="tile" href="https://fef.td.com/" target="_blank">
-              <img class="img-rounded" src="img/sponsor/td_friends.png"/>
-            </a>
-            <a class="tile" href="http://www.textreme.com/" target="_blank">
-              <img class="img-rounded" src="img/sponsor/textreme.png"/>
-            </a>
-            <a class="tile" href="https://www.engsoc.uwaterloo.ca/" target="_blank">
-              <img class="img-rounded" src="img/sponsor/uw_eng_soc.png"/>
-            </a>
-            <a class="tile" href="https://uwaterloo.ca/civil-environmental-engineering/" target="_blank">
-              <img class="img-rounded" src="img/sponsor/uw_science.png"/>
-            </a>
-            <a class="tile" href="http://newmediaservices.uwaterloo.ca/mediadoc.html" target="_blank">
-              <img class="img-rounded" src="img/sponsor/uw_media_doc.png"/>
-            </a>
-            <a class="tile" href="http://www.kitchenerhonda.com/en/" target="_blank">
-              <img class="img-rounded" src="img/sponsor/kitchener_honda.png"/>
-            </a>
-            <a class="tile" href="https://delta-elevator.com/" target="_blank">
-              <img class="img-rounded" src="img/sponsor/delta_elevator.png"/>
-            </a>
-            <a class="tile" href="http://odg.com/" target="_blank">
-              <img class="img-rounded" src="img/sponsor/ontario_drive_and_gear.png"/>
-            </a>
-            <a class="tile" href="http://www.teslamachining.com/" target="_blank">
-              <img class="img-rounded" src="img/sponsor/tesla_machining_and_gundrilling.png"/>
-            </a>
+            <div class="row">
+
+	            <div class="col-sm-2 col-sm-offset-1">
```

Comment:
here for example. the tab you are using isn't consistent with everywhere else in this file.